lte/protos/mconfig/mconfigs.proto

```diff
@@ -188,6 +188,9 @@ message SessionD {
     bool gx_gy_relay_enabled = 4;
     // sentry_config stores the Sentry.io configuration for this service
     SentryConfig sentry_config = 5;
+
+    // Enables 5G Standalone (SA) at a network level
+    bool enable5g_features = 6;
```

Comment:
you need to go to magma/orc8r/cloud/docker and run `./build.py -g` to generate the pb files

Reply:
Yeah added the auto generated file : magma/lte/cloud/go/protos/mconfig/mconfigs.pb.go